README.md

```diff
@@ -0,0 +1,43 @@
+# ICD10
+## Description
+This library main goal is to improve experience of using who icd 10 xml data by delivering a pack of helper methods. Due to licens we do not deliver the data itself, you have to get licence and data from official website https://www.who.int/classifications/help/FAQOther/en/.
```

Comment:
the experience 
using "who"? icd 10 xml? -> what's going on here? ;p
Due to license,
get a licence
the official website

Reply:
I had changed description a bit, hopefully now is more understable.